sky/clouds/utils/oci_utils.py

```diff
@@ -42,6 +44,9 @@ class OCIConfig:
     VCN_CIDR_INTERNET = '0.0.0.0/0'
     VCN_CIDR = '192.168.0.0/16'
     VCN_SUBNET_CIDR = '192.168.0.0/18'
+    SERVICE_PORT_RULE_TAG = 'SkyServe-Service-Port'
+    # NSG name format is prefix + clustername
+    NSG_NAME_PREFIX = 'nsg_'
```

Comment:
```suggestion
    NSG_NAME_TEMPLATE = 'nsg_{cluster_name}'
```
nit: can we use a template here? and using `NSG_NAME_TEMPLATE.format(cluster_name =xxx)` in other places.